.lighthouserc.js

```diff
@@ -26,7 +26,6 @@ module.exports = {
         `http://127.0.0.1:8181/create/${process.env.exploration_editor}`,
         `http://127.0.0.1:8181/collection_editor/create/${process.env.collection_editor}`,
         `http://127.0.0.1:8181/topic_editor/${process.env.topic_editor}`,
-        `http://127.0.0.1:8181/skill_editor/${process.env.skill_editor}`,
```

Comment:
Why delete?

Reply:
Do you have a follow-up tracking issue for this? Have you discussed this with the other members of the speed team (@vojtechjelinek and @jameesjohn)? 

Also, why does this not affect the topic editor etc.?

Reply:
I was able to make the setup script compatible with both the production and dev server by adding some conditional statements to the url collection process.  So I no longer need to remove it :)!